views/news/news-container.js

```diff
@@ -58,7 +58,13 @@ export default class NewsContainer extends React.Component {
       return null
     }
 
-    return <Image source={{uri: imageUrl}} style={styles.itemImage} />
+    let icon = <Image source={{uri: imageUrl}} style={styles.itemImage} />
+
+    Image.getSize(imageUrl, (width, height) => {
+      icon = (width >= 50 && height >= 50) ? icon : null
+    })
+
+    return icon
```

Comment:
This won't work correctly. `Image.getSize` is asynchronous – the callback may not (should not, I think) have executed by the time we return from the function.

We'll have to make `findImage` return a promise, instead, I think, and store some state in the component.